src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.cs

```diff
@@ -3324,46 +3324,144 @@ public new abstract class Arm64 : ArmBase.Arm64
         public static Vector64<float> DivideScalar(Vector64<float> left, Vector64<float> right) => DivideScalar(left, right);
 
         /// <summary>
-        ///  int8x8_t vmovn_s16 (int16x8_t a)
-        ///   A32: VMOVN.I16 Dd, Qm
-        ///   A64: XTN Vd.8B, Vn.8H
+        /// uint8_t vget_lane_u8 (uint8x8_t v, const int lane)
+        ///   A32: VMOV.U8 Rt, Dn[lane]
+        ///   A64: UMOV Wd, Vn.B[lane]
         /// </summary>
-        public static Vector64<sbyte> ExtractAndNarrowLow (Vector128<short> value) => ExtractAndNarrowLow (value);
+        public static byte Extract(Vector64<byte> vector, byte index) => Extract(vector, index);
 
         /// <summary>
-        ///  int16x4_t vmovn_s32 (int32x4_t a)
-        ///   A32: VMOVN.I32 Dd, Qm
-        ///   A64: XTN Vd.4H, Vn.4S
+        /// float64_t vget_lane_f64 (float64x1_t v, const int lane)
+        ///   A32: VMOV.F64 Dd, Dm
+        ///   A64: DUP Dd, Vn.D[lane]
         /// </summary>
-        public static Vector64<short> ExtractAndNarrowLow (Vector128<int> value) => ExtractAndNarrowLow (value);
+        public static double Extract(Vector64<double> vector, byte index) => Extract(vector, index);
```

Comment:
Not sure this API makes sense, `index` can only ever be `0`.

Reply:
It can probably just be covered by `vector.ToScalar()`?

Reply:
Or maybe just expose it as `public static double Extract(Vector64<double>)`?

Reply:
Actually, I was pretty sure that I removed this overload. I agree we don't need it at all

Reply:
How about keeping them and renaming as ?
```c#
public static double ExtractScalar(Vector64<double>);
public static long ExtractScalar(Vector64<long>);
public static ulong ExtractScalar(Vector64<ulong>);
```
So it would be more clear that these methods don't need `index` parameter?

cc @TamarChristinaArm @CarolEidt

Reply:
Hmm the name `ExtractScalar` does seem kind of inconsistent with our use of `Scalar` till now. We've only used it when the argument is a `scalar` so far, but in this case the intrinsics is also well defined for `vectors` as well.  And both use cases are valid and expected here.

It also doesn't tell you where it's extracting from. The extra parameter did serve as "documentation" here, so if it's removed I think the name should reflect how it's extracting the scalar. So maybe something like `ExtractLower`?

Reply:
Ah, I see, sorry in context (after expanding the diff) this makes sense. Yeah I'd be happy with both `Extract` and `ExtractScalar` with a slight preference to `Extract` for consistency with the rest of the instructions.

Reply:
Perhaps, these are also redundant and we, indeed, need should use ToScalar() and remove Extract for 64x1_t types?